docs/reference/search/field-stats.asciidoc

```diff
@@ -263,3 +263,25 @@ Each index constraint support the following comparisons:
 `gt`::  	Greater-than
 `lte`:: 	Less-than or equal to
 `lt`::  	Less-than
+
+Field stats index constraints on date fields optionally also accept a format option, that overrides the format being
```

Comment:
"Field stats index constraints on date fields optionally accept a format option, used to parse the constraint's value. If missing, the format configured in the field's mapping is used.